crates/contracts/src/alloy.rs

```diff
@@ -70,3 +70,58 @@ macro_rules! bindings {
         }
     };
 }
+
+pub mod networks {
+    pub const MAINNET: u64 = 1;
+    pub const GNOSIS: u64 = 100;
+    pub const SEPOLIA: u64 = 11155111;
+    pub const ARBITRUM_ONE: u64 = 42161;
+    pub const BASE: u64 = 8453;
+    pub const POLYGON: u64 = 137;
+    pub const AVALANCHE: u64 = 43114;
+    pub const BNB: u64 = 56;
+    pub const OPTIMISM: u64 = 10;
+    pub const LENS: u64 = 232;
+}
+
+crate::bindings!(GnosisSafe, maplit::hashmap! {});
+crate::bindings!(GnosisSafeCompatibilityFallbackHandler, maplit::hashmap! {});
+crate::bindings!(GnosisSafeProxy, maplit::hashmap! {});
+crate::bindings!(GnosisSafeProxyFactory, maplit::hashmap! {});
```

Comment:
The macro needs to be extended to support empty hashmaps, so the call could look like:
```suggestion
crate::bindings!(GnosisSafe);
crate::bindings!(GnosisSafeCompatibilityFallbackHandler);
crate::bindings!(GnosisSafeProxy);
crate::bindings!(GnosisSafeProxyFactory);
```

Reply:
Done